packages/next/next-server/lib/amp-context.ts

```diff
@@ -1,3 +1,4 @@
 import * as React from 'react'
 
 export const AmpStateContext: React.Context<any> = React.createContext({})
+AmpStateContext.displayName = 'AmpStateContext'
```

Comment:
Can you wrap these in NODE_ENV checks like so:

```
if(process.env.NODE_ENV !== 'production') {
  AmpStateContext.displayName = 'AmpStateContext'
}
```

This ensures production bundle size is not affected